android/app/build.gradle

```diff
@@ -44,6 +48,10 @@ dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     testImplementation 'junit:junit:4.12'
 
+    // Usb serial
+    implementation "org.jetbrains.anko:anko-common:0.8.3"

```

Comment:
Anko is [deprecated](https://github.com/Kotlin/anko), there should be alternatives.

Reply:
removed it (was added by mistake)